assets/l10n/app_en.arb

```diff
@@ -932,6 +932,18 @@
   "@errorMarkAsUnreadFailedTitle": {
     "description": "Error title when mark as unread action failed."
   },
+  "markAllAsReadConfirmationDialogTitle": "Mark all messages as read?",
+  "@markAllAsReadConfirmationDialogTitle": {
+    "description": "Title of the confirmation dialog for marking all messages as read"
+  },
+  "markAllAsReadConfirmationDialogMessage": "You are not in a conversation view. Marking all as read may unintentionally mark unseen messages as read. Do you want to continue?",
+  "@markAllAsReadConfirmationDialogMessage": {
+    "description": "Message that warns the user they may unintentionally mark messages as read"
+  },
+  "markAllAsReadConfirmationDialogAction": "Yes, mark as read",
+  "@markAllAsReadConfirmationDialogAction": {
+    "description": "Action message to confirm marking all messages as read."
+  },
```

Comment:
Please add entries to this file in the same commit as they are used in UI code, so the proposed change is presented as a coherent whole.